spp_country_office/views/country_office_views.xml

```diff
@@ -0,0 +1,117 @@
+                                        />
+                                        <field
+                                            name="street2"
+                                            placeholder="Street 2..."
+                                            class="o_address_street"
+                                        />
+                                        <field name="city" placeholder="City" class="o_address_city" />
+                                        <field
+                                            name="state_id"
+                                            class="o_address_state"
+                                            placeholder="State"
+                                            options='{"no_open": True}'
+                                        />
+                                        <field name="zip" placeholder="ZIP" class="o_address_zip" />
+                                        <field
+                                            name="country_id"
+                                            placeholder="Country"
+                                            class="o_address_country"
+                                            options='{"no_open": True, "no_create": True}'
+                                        />
+                                    </div>
+                                    <field name="vat" />
+                                    <field
+                                        name="currency_id"
+                                        options="{'no_create': True, 'no_open': True}"
+                                        id="company_currency"
+                                        context="{'active_test': False}"
+                                    />
+                                </group>
+                                <group>
+                                    <field name="phone" class="o_force_ltr" />
+                                    <field name="mobile" class="o_force_ltr" />
+                                    <field name="email" />
+                                    <field
+                                        name="website"
+                                        string="Website"
+                                        widget="url"
+                                        placeholder="e.g. https://www.odoo.com"
+                                    />
+                                </group>
+                                <group name="social_media" />
+                            </group>
+                        </page>
+                    </notebook>
+                </sheet>
+            </form>
+        </field>
+    </record>
+
+    <record id="view_country_office_tree" model="ir.ui.view">
+        <field name="name">view_country_office_tree</field>
+        <field name="model">res.company</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="name" string="Country Office" />
```

Comment:
dunno if a list view with only 1 field is a good idea, @reichie020212

Reply:
Added a new field